test/functional/test_framework/key.py

```diff
@@ -13,7 +13,7 @@
 import ctypes.util
 import hashlib
 
-ssl = ctypes.cdll.LoadLibrary(ctypes.util.find_library ('ssl') or 'libeay32')
+ssl = ctypes.cdll.LoadLibrary(ctypes.util.find_library ('ssl'))
```

Comment:
Why is this change necessary?

Reply:
this is leftover from my test environment. it usually complaints about library libeay32 not found when I run functional tests. I remove it as a shortcut and continue texting. it shouldn't be in the repo. will fix it.